### DISCUSSION TITLE
Week9：macOS 安裝 XAMPP 後遇到 Error starting "XAMPP" stack 的問題。

### DISCUSSION THREAD

**Discussion by dadidi910** [Lidemy/forum#136] [後端相關問題] [answered]
想詢問有沒有同學或是老師們可以協助解答這個問題呢？

<img width="871" alt="截圖 2021-06-07 下午2 25 58" src="https://user-images.githubusercontent.com/80995032/120969276-8c716780-c79c-11eb-92cb-321566ad1272.png">

```
ERROR: Error starting "XAMPP" stack: cannot calculate MAC address: Using fd 7 for I/O notifications hv_vm_create HV_ERROR (unspecified error)
```

我的版本是 macOS Big Sur 版本 11.4

查了一下資料找到了這個
https://github.com/Manoj-Athi/Xampp-error

因為看不懂怎麼使用 > < 想詢問是不是可以用這個方法解決呢？
有沒有其他同學遇到一樣的狀況呢？

謝謝大家 :)

**Comment by dadidi910** [comment, marked as answer]
已解決此問題～謝謝在群組協助回覆的老師和同學唷 : )

如有相同問題的同學，按照網址內的步驟即可解決問題唷！

https://stackoverflow.com/questions/67558760/xampp-problem-mac-address-and-hv-error-not-detected
這裡也可以參考唷！

**Reply by KW0409** [reply]
我自己是 Mac m1 系統的，之前安裝的時候也有碰到一樣的問題，但是這些解法對 M1 系統好像是無效的
後來有找到解法是看到 [這篇文章](https://community.apachefriends.org/f/viewtopic.php?f=29&t=80239&p=271269&hilit=installer+version#p271269)，**簡單來說就是不要下載 VM 版本的，改成下載 installer 版本的即可**(可 [點此連結](https://sourceforge.net/projects/xampp/files/XAMPP%20Mac%20OS%20X/8.0.12/) 查看下載)
不過之前忘了分享了，在這邊補上給大家參考
最近又找到這篇 [How to get Xampp working on your M1 chip Macbook pro](https://dev.to/deborahtrez/how-to-get-xampp-to-work-on-m1-macbook-pro-47gi)
意思也是下載 installer，但感覺講得比上面那篇會更清楚一點
